components/asana/asana.app.mjs

```diff
@@ -0,0 +1,429 @@
+import axios from "axios";
+
+export default {
+  type: "app",
+  app: "asana",
+  propDefinitions: {
+    organizations: {
+      label: "Organizations",
+      description: "List of organizations. This field use the organization GID.",
+      type: "string[]",
+      async options() {
+        const organizations = await this.getOrganizations();
+
+        return organizations.map((organization) => {
+          return {
+            label: organization.name,
+            value: organization.gid,
+          };
+        });
```

Comment:
This can be simplified.
```suggestion
        return organizations.map((organization) => ({
          label: organization.name,
          value: organization.gid,
        }));
```